plone/app/portlets/tests/test_formextender.py

```diff
@@ -0,0 +1,270 @@
+# -*- coding: utf-8 -*-
+
+from plone.app.portlets.browser.interfaces import (IPortletAddForm,
+                                                   IPortletEditForm)
```

Comment:
see Plone Style Guide https://docs.plone.org/develop/styleguide/python.html#about-imports
applies also below

Reply:
Thanks:) yep, my isort was missing the recommended plone config (i suggest we add this to the setup.cfg ? )